src/styles/blocks/card.scss

```
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
+  font-family: inherit;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description-name {
+    padding-top: 40px;
```

use margins to create distance BETWEEN elements
```suggestion
    margin-top: 40px;
```